CHANGELOG.md

```diff
@@ -4,9 +4,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## Unreleased
 ### Added
 
-- …
+- Initial implementation ([#1])
 
-[Unreleased]: https://github.com/projectsyn/component-nfs-client-provisioner/compare/v0.1.0...HEAD
```

Comment:
Why are you removing this link? It should point to https://github.com/projectsyn/component-nfs-client-provisioner/compare/1dee7be0a89165228756f70a106e8d8f62cbf5eb...HEAD